extensions/oidc/runtime/src/main/java/io/quarkus/oidc/SecurityEvent.java

```diff
@@ -70,6 +74,11 @@ public SecurityEvent(Type eventType, Map<String, Object> eventProperties) {
         this.eventType = eventType;
     }
 
+    public SecurityEvent(Type eventType) {
```

Comment:
Not sure it is necessary, since it is worth including an address of unavailable server

Reply:
I'll drop it.

Reply:
Done.